db/migrate/20160426145517_move_host_description_to_host_comment.rb

```diff
@@ -0,0 +1,29 @@
+class MoveHostDescriptionToHostComment < ActiveRecord::Migration
+  class System < ActiveRecord::Base
+    self.table_name = "katello_systems"
+  end
+
+  def up
+    System.find_each do |system|
```

Comment:
I would base this on ::Host rather than System.  This works, but Host is better i'd say.  No need to involve system here.